CUDADataFormats/TrackingRecHit/interface/TrackingRecHit2DSOAView.h

```diff
@@ -0,0 +1,101 @@
+#ifndef CUDADataFormats_TrackingRecHit_interface_TrackingRecHit2DSOAView_h
+#define CUDADataFormats_TrackingRecHit_interface_TrackingRecHit2DSOAView_h
+
+#include <cuda_runtime.h>
+
+#include "CUDADataFormats/SiPixelCluster/interface/gpuClusteringConstants.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+#include "Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h"
+
+namespace pixelCPEforGPU {
+  struct ParamsOnGPU;
+}
+
+class TrackingRecHit2DSOAView {
+public:
```

Comment:
this looks particularly coupled to the pixel Phase1, while the name of the class is generic.
Better put PixelPhase1 in the name or outline how this can work for Phase-2 and also for strips.

Reply:
Given the ongoing and proposed work to make the pixel code more generic, I would propose to avoid any renaming for the time being.
Once the Phase 2 support lands, we can add Phase1 / Phase2 to the class names as needed, while keeping the common classes untouched.